app/models/infra_conversion_job.rb

```diff
@@ -1,9 +1,5 @@
 class InfraConversionJob < Job
   def self.create_job(options)
```

Comment:
May as well just remove the method if there's nothing in it.

Reply:
There's not nothing. There's `super(name, options)`. And without it, it fails.
Other subclasses of Job use it too.

Reply:
Yeah, IMO that is a _really_ weird API that `Job.create_job` exposes.

The only time that would make sense is if we wanted to call `Job.create_job("Klass", options)` instead of `Klass.create_job(options)` for some reason.  It looks like this is actually what we do for [VmScan](https://github.com/ManageIQ/manageiq/blob/master/app/models/vm_or_template/scanning.rb#L32) though I'm not sure why `VmScan.create_job(options)` wouldn't work in that case.

Either way, yes unfortunately this is needed but I would love to clean this up in the future.

Reply:
@agrare Are you thinking just having the first argument default to `name`?

Reply:
Think we can drop name completely and only have people call create_job on the leaf classes.  Then the base models create_job would just use name